docs/pages/upgrade-existing-vclusters.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Upgrade existing vclusters
+sidebar_label: Upgrade existing vclusters
+---
+
+In this section, you will find steps and guidelines to upgrade/downgrade existing vclusters. For all kinds of kubernetes distros
+supported by vcluster (k0s, k3s, k8s and eks), the latest image information can be found on the links below.
+
+If you want to contribute to vcluster by upgrading kubernetes distros images directly, PR - https://github.com/loft-sh/vcluster/pull/666 can be referred to know what files need to be changed.
+
+It's very easy to update the existing vclusters running on your host clusters. You just need to update the statefulset or deployment depending on your vcluster distro.
```

Comment:
Why have you decided to document upgrades through statefulset/deployment updates and not via `vcluster create --upgrade` CLI command? For the CLI command, we can set `--kubernetes-version` flag, and `--chart-version` to choose versions of k8s and syncer respectively. Or for more fine-grained control, the exact images can be defined via a values file.